Jenkinsfile

```diff
@@ -101,6 +102,23 @@ node('ubuntu-zion') {
         gitHub.statusUpdate commitId, 'success', 'test', 'Tests succeeded'
       }
     }
+    stage('Evaluate') {
+      
+      //Create tar of our image
+      OsTools.runSafe(this, "docker save ${imageName} -o ${env.WORKSPACE}/${tarName}")
+      
+      //decide which stage we are creating
+      def theStage = 'build'
+      if(branch == 'master') {
+        theStage = 'release'
+      }
+
+      //run the evaluation
+      nexusPolicyEvaluation iqStage: "${theStage}", iqApplication: "${archiveName}",
```

Comment:
I don't object to using "docker-nexus-iq-server" as the IQ app id but pulling that value from a variable named "archiveName" which is otherwise used well for the tarball seems dangerous: Somebody later tweaking that value above at the beginning of the script might be surprised to find this affected policy evaluation as well... Hence I suggest to introduce a dedicated variable `iqApplicationId` or similar which can be set close to https://github.com/sonatype/docker-nexus-iq-server/blob/7e18ade46aa03bf7d9db34ea664e27fb400b553d/Jenkinsfile#L28 where all the other variables are initialized.

Reply:
Added new variable in b6aaba7